CI build tests failed on an unrelated (reported in https://github.com/zephyrproject-rtos/zephyr/issues/106462):
```
INFO    - 180/944 stm32wb5mmg/stm32wb55xx   application_development.ram_context_for_isr        ERROR Build failure (build <zephyr/gnu>)
(...)
gen_isr_tables.py: error: multiple registrations at table_index 40 for irq 40  (0x28)
Existing handler 0x20000021, new handler 0x80026e5
Has IRQ_CONNECT or IRQ_DIRECT_CONNECT accidentally been invoked on the same irq multiple times?

ninja: build stopped: subcommand failed.
(...)
```
I've append a commit for propose to exclude STM32 SoCs from this test since not easily applicable.

--- a/boards/st/nucleo_wb55rg/nucleo_wb55rg.yaml
+++ b/boards/st/nucleo_wb55rg/nucleo_wb55rg.yaml
@@ -6,7 +6,7 @@ toolchain:
   - zephyr
   - gnuarmemb
 ram: 192
-flash: 808
+flash: 876
 supported:
   - adc
   - arduino_gpio

--- a/boards/st/stm32wb5mmg/stm32wb5mmg.yaml
+++ b/boards/st/stm32wb5mmg/stm32wb5mmg.yaml
@@ -6,7 +6,7 @@ toolchain:
   - zephyr
   - gnuarmemb
 ram: 192
-flash: 808
+flash: 876
 supported:
   - gpio
   - dma

--- a/dts/arm/st/wb/stm32wb55Xg.dtsi
+++ b/dts/arm/st/wb/stm32wb55Xg.dtsi
@@ -14,8 +14,12 @@
 	soc {
 		flash-controller@58004000 {
 			flash0: flash@8000000 {
-				reg = <0x08000000 DT_SIZE_K(808)>;
-				ranges = <0 0x8000000 DT_SIZE_K(808)>;
+				/* Physical flash size is 1MByte but the last 148kByte
+				 * are assigned to the wireless firmware running on
+				 * Cortex-M0 companion.
+				 */
+				reg = <0x08000000 DT_SIZE_K(876)>;
+				ranges = <0 0x8000000 DT_SIZE_K(876)>;
 			};
 		};
 	};

